### PR TITLE
Initial API specs

### PR DESCRIPTION
**Pull request by rameshmalla**
#1

### CODE REVIEW

File: api/api.yaml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+      status:
+        $ref: '#/definitions/MissionStatus'
+      sprint:
+        type: string
+        example: "Sprint 1"
+      team:
+        type: string
+        example: "team-id-123"
+      crtical:
+        $ref: '#/definitions/Crtitcality'
```
Comment (prashanth-92) [inline api/api.yaml]: spelling mistake

File: api/api.yaml [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        description: "true if the mission is still un explored (not yet worked on)"
+      status:
+        $ref: '#/definitions/MissionStatus'
+      sprint:
+        type: string
+        example: "Sprint 1"
+      team:
+        type: string
+        example: "team-id-123"
+      crtical:
```
Comment (prashanth-92) [inline api/api.yaml]: Critical spelling mistake

File: api/api.yaml [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+      - id
+      - name
+      - description
+      - created_by
+      - team
+      - created_at
+      - updated_at
+      
+  
+  Crtitcality:
```
Comment (prashanth-92) [inline api/api.yaml]: Criticality spelling mistake

File: api/api.yaml [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+        example: "Error description"
+      
+parameters:
+  team-id:
+    name: team-id
+    in: path
+    type: string
+    required: true
+    
+  sales_channel:
```
Comment (prashanth-92) [inline api/api.yaml]: remove this. It is unnecessary

File: api/api.yaml [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+          type: string
+        - name: "mission"
+          in: body
+          description: "Mission request"
+          schema:
+            $ref: '#/definitions/Mission'
+     responses:
+        200:
+          description: "Successful operation"
+          schema:
```
Comment (prashanth-92) [inline api/api.yaml:118]: PUT need not return anything, i feel!

Comment (rameshmalla) [reply api/api.yaml:118]: I think returning 204, would result in another network call to get the updated content.

File: api/api.yaml (hunk 1 above)
Comment (RameshMalla) [reply api/api.yaml]: 👍

**Review by prashanth-92 — APPROVED** [review]

**Pull request merged**